### ISSUE TITLE
Webpack loader imports fragment multiple times when querying an interface

### ISSUE THREAD

**Issue by clayne11** [apollographql/graphql-tag#74]
I'm having an issue where the webpack loader loads the fragment multiple times in certain instances.

This is my setup:

```graphql
# schema
interface Foo {
  bar: String!
}

type FooBar implements Foo {
  bar: String!
}

type OtherFooBar implements Foo {
  bar: String!
}



# Query.graphql
#import 'FragmentOne.graphql'
#import 'FragmentTwo.graphql'
query Test {
  ...FragmentOne
  ...FragmentTwo
}



# FragmentOne.graphql
#import 'FragmentThree.graphql'

fragment FragmentOne on FooBar {
  ...FragmentThree
}



# FragmentTwo.graphql
#import 'FragmentThree.graphql'

fragment FragmentTwo on OtherFooBar {
  ...FragmentThree
}



# FragmentThree.graphql
fragment FragmentThree on Foo {
  bar
}
```

Result:
```graphql
query Test {
  ...FragmentOne
  ...FragmentTwo
}

fragment FragmentOne on FooBar {
  ...FragmentThree
}

fragment FragmentThree on Foo {
  bar
}

fragment FragmentTwo on OtherFooBar {
  ...FragmentThree
}

fragment FragmentThree on Foo {
  bar
}
```

Expected result (with de-duping):
```graphql
query Test {
  ...FragmentOne
  ...FragmentTwo
}

fragment FragmentOne on FooBar {
  ...FragmentThree
}

fragment FragmentTwo on OtherFooBar {
  ...FragmentThree
}

fragment FragmentThree on Foo {
  bar
}
```

`FragmentThree` is being included multiple times which is throwing an error since each fragment is only allowed to be defined once.

**Comment by clayne11** [comment]
Looks like this is a duplicate of #27 and is fixed by #52. Is #52 going to be merged any time soon?

**Comment by czert** [comment]
@clayne11 my bad, #52 still needs some work before merging. The fix in my fork works well enough though, so you can use it until I get around to finishing it.

**Comment by clayne11** [comment]
I am using it, definitely looking forward to it landing.

**Comment by jnwng** [comment]
going to close as a duplicate of #27 fixed by #52
